pkgs/top-level/python-packages.nix

```diff
@@ -257,6 +259,8 @@ in {
     pythonPackages = self;
   };
 
+  pysc2 = callPackage ../development/python-modules/pysc2 { };
```

Comment:
Is this supposed to be used as a library or a command line application?

Reply:
Both. The package comes with scripts for running a machine learning agent as a player an a Starcraft 2 scenario, but it is also a library for creating such an agent.